crates/uv-cache-info/src/glob.rs

```diff
@@ -0,0 +1,293 @@
+use std::{
+    collections::BTreeMap,
+    path::{Component, Components, Path, PathBuf},
+};
+
+fn is_glob_like(part: Component) -> bool {
+    matches!(part, Component::Normal(_))
+        && part
+            .as_os_str()
+            .to_str()
+            .is_some_and(|part| ["*", "{", "}", "?"].into_iter().any(|c| part.contains(c)))
```

Comment:
I think this needs to check for `[` and `]` too. I think [that's it](https://docs.rs/globset/latest/src/globset/lib.rs.html#944-961).

Reply:
Done.